modules/core/src/convert.cpp

```diff
@@ -5287,6 +5292,43 @@ static bool ocl_LUT(InputArray _src, InputArray _lut, OutputArray _dst)
 
 #endif
 
+#ifdef HAVE_OPENVX
+static bool openvx_LUT(Mat src, Mat dst, Mat _lut)
+{
+    if (src.type() != CV_8UC1 || dst.type() != src.type() || _lut.type() != src.type() || !_lut.isContinuous())
+        return false;
+
+    try
+    {
+        ivx::Context ctx = ivx::Context::create();
+
+        ivx::Image
+            ia = ivx::Image::createFromHandle(ctx, VX_DF_IMAGE_U8,
+                std::vector<vx_imagepatch_addressing_t>(1, ivx::Image::createAddressing(src.cols, src.rows, 1, (vx_int32)(src.step))),
+                std::vector<void*>(1, src.data)),
+            ib = ivx::Image::createFromHandle(ctx, VX_DF_IMAGE_U8,
+                std::vector<vx_imagepatch_addressing_t>(1, ivx::Image::createAddressing(dst.cols, dst.rows, 1, (vx_int32)(dst.step))),
+                std::vector<void*>(1, dst.data));
+
+        ivx::LUT lut = ivx::LUT::create(ctx);
+        lut.copyFrom(_lut);
+        ivx::IVX_CHECK_STATUS(vxuTableLookup(ctx, ia, lut, ib));
+    }
+    catch (ivx::RuntimeError & e)
+    {
+        CV_Error(CV_StsInternal, e.what());
+        return false;
+    }
+    catch (ivx::WrapperError & e)
+    {
+        CV_Error(CV_StsInternal, e.what());
+        return false;
+    }
```

Comment:
Looks like we need a base class `ivx::Error` to prevent duplicated code for both types of ivx exceptions...